dashboard/db/migrate/20230313222451_backfill_regional_partner_is_active.rb

```diff
@@ -0,0 +1,6 @@
+class BackfillRegionalPartnerIsActive < ActiveRecord::Migration[6.0]
+  def change
+    RegionalPartner.reset_column_information
+    RegionalPartner.update_all(is_active: true)
+  end
```

Comment:
this looks like something you only want to do if you are running the migration forward, but not if you are rolling it back, correct? if so, this method should be be renamed to `up`, and there should also be an empty `down` method.

Reply:
Yes, that is correct.

I was actually curious about that and especially on using up/down vs change. I [learnt that](https://www.gapintelligence.com/blog/up-and-down-a-rails-migration/) in more recent versions of rails, it is can deduce the up/down from the change operation and this was a preferable option to use.

I also tested this in both migrate and roll back operations and found that roll back was a no-op. I like the suggestion of using up/down for overall readability and being very explicit about the roll back.